website/content/docs/commands/license/inspect.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: license get - Command
+description: |-
+  The "license get" command reports on the license in use.
```

Comment:
This should reference `license inspect` instead of `license get`. Same with the page title.

Reply:
Fixed